activity.py

```diff
@@ -245,7 +249,7 @@ def __init__(self, handle):
         self._chat = chat.View()
         self._chat.show_all()
         self._notebook.append_page(self._chat, Gtk.Label(''))
-
+        
```

Comment:
Also look at the diff of your change before committing to avoid changes like this.